conda_forge_tick/auto_tick.py

```diff
@@ -477,6 +482,20 @@ def migration_factory(
         )
 
 
+def create_migration_yaml_creator(migrators: MutableSequence[Migrator], gx: nx.DiGraph):
+    with indir("../conda-forge-pinning-feedstock/recipe"):
+        pinnings = parse_config_file('conda_build_config.yaml',
+                                     config=Config(**CB_CONFIG))
+    # exclude non-package information
+    for k, v in pinnings.items():
+        # check if an actual package
+        if k in gx.nodes:
+            current_pins = map(str, v)
+            current_version = str(gx.nodes[k]['payload']['version'])
+            if VersionOrder(current_version) > max(map(VersionOrder(current_pins))):
+                migrators.append(MigrationYamlCreator(k, current_version))
```

Comment:
This block could handle multiple outputs pretty easily.

Reply:
What do you mean "multiple outputs"?

Reply:
For example, the root recipe has more than one output, only one of which is pinned (root_base).

Reply:
Hmm ok, I need to bubble up the max pin better.